src/ddmd/cond.d

```diff
@@ -508,12 +508,20 @@ extern (C++) final class StaticIfCondition : Condition
                 printf("\ts = '%s', kind = %s\n", sds.toChars(), sds.kind());
             }
         }
+
+        int errorReturn()
```

Comment:
Essentially, `error` (aka `errorReturn`) needs a context pointer, in order to access class instance members, even though the compiler could essentially inlined the code. Kenji worked on and fixed [many related issues](https://issues.dlang.org/buglist.cgi?bug_status=UNCONFIRMED&bug_status=NEW&bug_status=ASSIGNED&bug_status=REOPENED&bug_status=RESOLVED&bug_status=VERIFIED&bug_status=CLOSED&component=dmd&email1=k.hara.pg%40gmail.com&emailassigned_to1=1&emailcc1=1&emaillongdesc1=1&emailreporter1=1&emailtype1=substring&list_id=214276&product=D&query_format=advanced&resolution=---&resolution=FIXED&resolution=INVALID&resolution=WONTFIX&resolution=LATER&resolution=REMIND&resolution=DUPLICATE&resolution=WORKSFORME&resolution=MOVED&short_desc=nested%20context%20closure&short_desc_type=anywordssubstr) in the compiler, but since then, no one seems to have picked up where he left. There are still [many issues](https://issues.dlang.org/buglist.cgi?bug_status=UNCONFIRMED&bug_status=NEW&bug_status=ASSIGNED&bug_status=REOPENED&component=dmd&list_id=214277&product=D&query_format=advanced&resolution=---&short_desc=alias%20nested%20context%20closure&short_desc_type=anywordssubstr) left in this area.